docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -1604,6 +1604,67 @@ public class OidcStartup {
 For more complex setup involving multiple tenants please see the xref:security-openid-connect-multitenancy.adoc#programmatic-startup[Programmatic OIDC start-up for multitenant application]
 section of the OpenID Connect Multi-Tenancy guide.
 
+== Step Up Authentication
+
+The `io.quarkus.oidc.AuthenticationContext` annotation can be used to enforce required authentication level for the Jakarta REST resource classes and methods.
```

Comment:
Also, since it can also be effective for WebSockets Next, may be we should just say `to enforce a required authentication level` and leave it that ? FYI, the annotation itself just says `endpoints` without listing the actual technologies

Reply:
This is similar situation to what we have for `@Tenant`, I am bit worried to avoid mentioning Jakarta REST and WS Next because users have no means to figure if it works for gRPC / GraphQL / MCP Server etc. (I suppose it doesn't). I got what you mean and maybe it can be better phrased, but I think we need to be explicit.

Reply:
You can suggest alternative way to say that though.

Reply:
@michalvavrik Sounds good, let's keep it as is for now